packages/rest/src/lib/CDN.ts

```diff
@@ -8,19 +8,48 @@ import {
 	StickerExtension,
 } from './utils/constants';
 
+/**
+ * The options used for image URLs
+ */
 export interface BaseImageURLOptions {
+	/**
+	 * The extension to use for the image URL
+	 * @default 'webp'
+	 */
 	extension?: ImageExtension;
+	/**
+	 * The size specified in the image URL
+	 */
 	size?: ImageSize;
 }
 
+/**
+ * The options used for image URLs with dynamic content
```

Comment:
This might be better wording now that `dynamic` is no more:
```suggestion
 * The options used for image URLs with animated content
```